src/pages/index.tsx

```diff
@@ -1,3 +1,62 @@
+import { Post } from "@/shared/components/index";
+import { useEffect, useState } from "react";
+import styles from "@/pages/index.module.scss";
+import instance from "./api/axiosInstance";
+
+type Res = {
+  item: {
+    id: string;
+    hourlyPay: number;
+    startsAt: string;
+    workhour: number;
+    description: string;
+    closed: boolean;
+    shop: {
+      item: {
+        id: string;
+        name: string;
+        category: string;
+        address1: string;
+        address2: string;
+        description: string;
+        imageUrl: string;
+        originalHourlyPay: number;
+      };
+    };
+  };
+};
+
+type Props = {
+  item: Res[];
+};
+
 export default function Home() {
-  return <div>메인페이지</div>;
+  const [userData, setUserData] = useState<Props["item"]>([]);
+  useEffect(() => {
+    const fetchData = async () => {
+      const response = await instance.get(`notices`);
+      setUserData(response.data.items);
+    };
+    fetchData();
+  }, []);
+
+  return (
+    <div>
+      {userData.map((data: Res) => (
+        <div className={styles.a} key={data.item.id}>
+          <Post
+            imageUrl={data.item.shop?.item?.imageUrl}
+            key={data.item.id}
+            startsAt={data.item.startsAt}
+            workhour={data.item.workhour}
+            hourlyPay={data.item.hourlyPay}
+            closed={data.item.closed}
+            name={data.item.shop?.item?.name}
+            address1={data.item.shop?.item?.address1}
+            originalHourlyPay={data.item.shop?.item?.originalHourlyPay}
+          />
+        </div>
+      ))}
```

Comment:
props가 많은 경우에는 스프레드 연산자로 표현해줘도 좋습니다!
```suggestion
      {userData.map((data: Res) => (
        <div className={styles.a} key={data.item.id}>
          <Post
            key={data.item.id}
            {...data}
          />
        </div>
      ))}
```